homeassistant/components/sensor/flunearyou.py

```diff
@@ -142,7 +144,8 @@ def state(self):
     @property
     def unique_id(self):
         """Return a unique, HASS-friendly identifier for this entity."""
-        return '{0}_{1}'.format(self._identifier, self._kind)
+        return '{0},{1}_{2}'.format(
+            self.fny.latitude, self.fny.longitude, self._kind)
```

Comment:
Is this a breaking change or did `self._identifier` just default to the value here?

Also, if `latitude` and `longitude are floating point numbers, they should be rounded to a known number of digits. Otherwise floating point arithmetic could accidentally create non-unique IDs. Look at the WUnderground sensor platform for inspiration.

Reply:
Ah nevermind about that first part (https://github.com/home-assistant/home-assistant/pull/19341/files#diff-ae3ae3cf093aa98d54e75b6e91e8b4bcL82 for reference)

Reply:
Re: your second point, there shouldn’t be any arithmetic actually occurring...unless string interpolation counts?

Reply:
Well how do you think floats are converted to a decimal representation 🙈 (Hint: [Using](https://github.com/python/cpython/blob/master/Objects/floatobject.c#L316) [floating](https://github.com/python/cpython/blob/4c49da0cb7434c676d70b9ccf38aca82ac0d64a9/Python/pystrtod.c#L801) [point](https://github.com/python/cpython/blob/e42b705188271da108de42b55d9344642170aa2b/Python/mysnprintf.c#L41) [division](https://linuxgazette.net/144/misc/lg/a_question_of_rounding_in_issue_143.html))

But then I read this comment: https://github.com/python/cpython/blob/4c49da0cb7434c676d70b9ccf38aca82ac0d64a9/Python/pystrtod.c#L833-L838 - python is smart enought to make this consistent by default :)

Reply:
Got it. Thanks!